client/containers/DashboardOverview/OverviewControls.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
```

Comment:
I am wondering about the benefit of unifying the controls for community- and collection-level scopes into a single component, when the different scopes have very different needs. Right now there are no community-level controls and that scope is best served by passing in a single "Create Pub" button — I've been imagining developing a `<ScopedCreatePubButton>` component for this purpose. Over time, though, we could add some — and this component would be full of conditionals to accomodate two mostly disjoint sets of controls. Really, the biggest thing "community overview controls" and "collection overview controls" have in common is that they happen to exist in the same place in the UI, and that's not something this component itself is aware of — so I'd advocate for making them two separate components.

Zooming out a little, I see these two overview screens being best served by two different components, rather than one component with a lot of conditionals, and achieving reuse through composition.

Reply:
Agreed. Fixed with refactor in latest commit.